app/controllers/team_repositories_controller.rb

```diff
@@ -20,9 +22,15 @@ def create
 
   # DELETE :team_id/repositories/:repository_id/team_repositories/:id
   def destroy
-    team_repository = @repository.team_repositories.find(destory_params[:id])
-    team_repository.destroy
-    render json: { message: 'destroyed!' }, status: :no_content
+    team_repository = @repository.team_repositories.find_by_id(destory_params[:id])
+
+    if team_repository
+      log_activity(:unshare_inventory, team_repository)
+      team_repository.destroy
+      render json: { message: 'destroyed!' }, status: :no_content
```

Comment:
204 response (`:no_content`) by the definition should not contain a message body.